ios/ByteowlsCapacitorOauth2/Source/ByteowlsCapacitorOauth2.swift

```diff
@@ -275,9 +284,12 @@ public class OAuth2ClientPlugin: CAPPlugin {
                     )
                 }
 
-                let urlHandler = SafariURLHandler(viewController: (bridge?.viewController)!, oauthSwift: oauthSwift)
-                // if the user touches "done" in safari without entering the credentials the USER_CANCELLED error is sent #71
-                urlHandler.delegate = self.oauth2SafariDelegate
+                let urlHandler = ASWebAuthenticationURLHandler(
+                    callbackUrlScheme: redirectUrl.replacingOccurrences(of: "://oauth", with: ""),
```

Comment:
Should this be:
```
callbackUrlScheme: URL(string: redirectUrl)!.scheme!,
```
So it works with any host url? e.g ://auth

Reply:
Yes, it should be as you've written. I've not tested it with other host urls but it should with any url.